compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1774,11 +1774,11 @@ bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode
       {
       case TR::vadd:
       case TR::vsub:
-         if (dt == TR::Int16 || dt == TR::Int8 || dt == TR::Int32 || dt == TR::Float || dt == TR::Double)
+         if (dt == TR::Int64)
```

Comment:
I don't think is right. I believe the vadd evaluator only works for Power 8 or later. This change will bypass the check earlier in the function.

Reply:
I meant to say the Int64 vadd evaluator only works for Power 8 or later. But, it looks like you got what I meant anyways.